frontend/src/static/css/_extra.css

```
@@ -3112,19 +3112,19 @@ body.dark_theme .profile-page-header .profile-info h1 {
 }
 
 body.dark_theme .profile-page-header .profile-nav {
-	background-color: #0b3144;
-	border-bottom-color: #0b3144;
+	background-color: var(--site-featured-bg-color);
```

Stylelint (17.11.1)

[error] Duplicate property "background-color" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)